Lil-Peep/src/styles/modules/site-header.scss

```diff
@@ -44,34 +54,162 @@
         justify-content: flex-end;
         width: 50%;
       }
+    }
+  }
 
-      .customer-login {
-        li {
-          display: flex;
-          margin-right: 20px;
-        }
+  .customer-login {
+    li {
+      display: flex;
+      margin-right: 20px;
+    }
 
-        ul {
-          display: flex;
-          margin-right: 20px;
-        }
+    ul {
+      display: flex;
+    }
 
-        .site-logo {
-          display: flex;
+    .site-logo {
+      display: flex;
 
-          img {
-            display: block;
-            justify-content: center;
-            width: 100%;
-          }
+      img {
+        display: block;
+        justify-content: center;
+        width: 100%;
+      }
+    }
+  }
+
+  .navigation {
+    display: flex;
+
+    .nav-list {
+      padding-left: 10%;
+      width: 80%;
+    }
+
+    .cart-icon {
+      width: 20%;
+      padding-left: 0;
+    }
+
+    .site-nav {
+      display: flex;
+      justify-content: space-evenly;
+      list-style: none;
+
+      a {
+        color: $color-categories-text;
+        text-decoration: none;
+
+        &:hover {
+          color: $color-text-hover;
+          text-decoration: underline;
         }
       }
     }
   }
+}
+
+@media only screen and (min-width: 769px) and (max-width: 1024px) {
+  .sidenav {
+    display: none;
+  }
+
+  .navbar {
+    display: none;
+  }
+
+  .header-mobile-section {
+    display: none;
+  }
+
+  .section-header {
+    padding: 15px;
+  }
+
+  header {
+    display: flex;
+    margin-top: 21px;
+    margin-bottom: 21px;
+
+    .header-logo {
+      display: flex;
+      justify-content: center;
+      width: 50%;
+      margin-top: 0;
+      margin-bottom: 0;
+    }
+
+    .h1 {
+      display: flex;
+      justify-content: center;
+      width: 50%;
+    }
+
+    a {
+      color: $color-categories-text;
+      text-decoration: none;
+    }
+
+    .site-logo {
+      left: 90%;
+      position: relative;
+    }
+
+    .site-header__logo-image {
+      left: 50%;
+      position: relative;
+    }
+  }
+
+  .searchlogin {
+    align-items: center;
+    display: flex;
+    justify-content: flex-end;
+    width: 50%;
+
+    #Search {
```

Comment:
Avoid using id selectors<br>Selector `Search` should be written in lowercase with hyphens